### DISCUSSION TITLE
Encrypt bounced emails before storing them on S3

### DISCUSSION THREAD

**Discussion by IndyV** [simple-login/app#604] [Feature Requests]
Currently bounced emails are [stored in an S3 bucket](https://simplelogin.io/security/#database--file-storage). While they are automatically deleted after 7 days they are not encrypted.
Meaning all bounced emails are passed to AWS in a readable format and could potentionally be read if exploited on Amazon's side.

It'd give privacy-focused users more peace of mind if the bounced emails are encrypted on the server side before being sent to the S3 bucket.

**Comment by strawrhu** [comment]
This is a good point and a feature that most users are unaware of.
